src/api/record/record.spec.ts

```diff
@@ -234,7 +234,7 @@ describe('Record Handler', function() {
     beforeEach(function() {
       recordMap['tape-1'] = Record.fromObj(MOCK_RECORD_1);
       let associateFs = {
-        '/var/incoming': {'path1': 'Video 1', 'path2': 'Video 2'},
+        '/var/archives/incoming': {'path1': 'Video 1', 'path2': 'Video 2'},
         './data/.db.json': '{}'
```

Comment:
There are half a dozen ways to configure it. When you install it for Q2, you have it in /mnt/archives; the program assumes `/var/archives` for testing (`/var` is commonly used in *nix for runtime data files, `/mnt` assumes the program is running somewhere with mounted file systems, which we don't need to do).